### PR TITLE
feat(superset): Add patch that fixes saved queries export

### PR DESCRIPTION
**Pull request by sbernauer**
# Description

Patch is taken from https://github.com/apache/superset/pull/26889.

Ideally I would like to wait with merge until the linked PR is merged

## Definition of Done Checklist

- Not all of these items are applicable to all PRs, the author should update this template to only leave the boxes in that are relevant
- Please make sure all these things are done and tick the boxes
 
```[tasklist]
- [ ] Changes are OpenShift compatible
- [ ] All added packages (via microdnf or otherwise) have a comment on why they are added
- [ ] Things not downloaded from Red Hat repositories should be mirrored in the Stackable repository and downloaded from there
- [ ] All packages should have (if available) signatures/hashes verified
- [ ] Does your change affect an SBOM? Make sure to update all SBOMs
```

### CODE REVIEW

**Review by NickLarsenNZ — DISMISSED** [review]
clever

File: superset/Dockerfile [hunk 1]
```diff
@@ -17,6 +17,7 @@ RUN microdnf update \
     && microdnf install \
         --assumeyes \
         cyrus-sasl-devel \
+        findutils \
```
Comment (lfrancke) [inline superset/Dockerfile:20]: Add a comment what this is needed for please

**Review by lfrancke — CHANGES_REQUESTED** [review]

File: superset/Dockerfile (hunk 1 above)
Comment (lfrancke) [reply superset/Dockerfile:21]: Yes!

File: superset/Dockerfile [hunk 2]
```diff
@@ -17,6 +17,8 @@ RUN microdnf update \
     && microdnf install \
         --assumeyes \
         cyrus-sasl-devel \
+        # Needed to find all patch files
```
Comment (lfrancke) [inline superset/Dockerfile:20]:
```suggestion
        # Needed to find all patch files, used in `apply_patches.sh`
```

**Review by lfrancke — APPROVED** [review]
One nit, looks good otherwise

**Review by lfrancke — APPROVED** [review]

**Comment by lfrancke** [comment]
Just for documentation purposes: Can you briefly say which version of Superset you expect this to be released in?

**Comment by sbernauer** [comment]
I'm not entirely sure, but I think it will be in the next patch releases of 3.0.x and 3.1.x

**Pull request merged**